commands/bootstrap/index.js

```diff
@@ -34,7 +34,15 @@ class BootstrapCommand extends Command {
   }
 
   initialize() {
-    const { registry, rejectCycles, npmClient = "npm", npmClientArgs, mutex, hoist } = this.options;
+    const {
+      registry,
+      rejectCycles,
+      npmClient = "npm",
+      npmClientArgs,
+      mutex,
+      hoist,
+      npmCiMode,
```

Comment:
I think we should take advantage of the existing (hidden) `--ci` flag, which is set automatically when `lerna` is called in a CI env (Travis, AppVeyor, etc). It would only take effect when `npmClient === "npm"`, of course, and would be passed to `npmInstall()` as `this.npmConfig.subCommand` (defaulting to `"install"`).